index.html

```diff
@@ -11,9 +11,12 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button type="button" class="menu-bars menu-bars--active">
         <i class="fas fa-bars"></i>
       </button>
+      <button type="button" class="menu-x">
+        <i class="fas fa-times"></i>
+      </button>
       <nav id="desktop-nav">
         <ul>
           <li><a class="nav-link" href="#projects">Portoflio</a></li>
```

Comment:
- [ ] [OPTIONAL] There is an issue with the links for the desktop version, since the image is placed on the HTML, it overlays part of the `<header>`, and it's not possible to click the links. I would suggest using the image as a background on the CSS with the property `background: url('IMAGE_URL');`. It is also a better practice to place background images on the CSS.

![image](https://user-images.githubusercontent.com/81522435/136108773-a79eedc2-b0fc-4af7-80df-b15a9bf27048.png)